### PR TITLE
docs: Link to the Alpine Linux community package for Synapse

### PR DESCRIPTION
**Pull request by 6543**

### CODE REVIEW

File: docs/setup/installation.md [hunk 1]
```diff
@@ -155,6 +155,14 @@ sudo pip uninstall py-bcrypt
 sudo pip install py-bcrypt
 ```
 
+#### AlpineLinux
+
+In the AlpineLinux packages exist [official maintained package](https://pkgs.alpinelinux.org/packages?name=synapse&branch=edge), so you can just install it with:
+
+```sh
+sudo apk add synapse
+```
```
Comment (DMRobertson) [inline docs/setup/installation.md:164]:
```suggestion
#### Alpine Linux

6543 maintains [Synapse packages for Alpine Linux]( (https://pkgs.alpinelinux.org/packages?name=synapse&branch=edge) in the community repository. Install with:

```sh
sudo apk add synapse
```

**Review by DMRobertson — CHANGES_REQUESTED** [review]
Could you also add a changelog file please?

File: changelog.d/16304.doc [hunk 2]
```diff
@@ -0,0 +1 @@
+Add alpine-linux package as official install option
```
Comment (DMRobertson) [inline changelog.d/16304.doc:1]:
```suggestion
Link to the Alpine Linux community package for Synapse.
```

(Sorry, I'd do make this change myself but it seems that maintainers aren't granted edit rights to your PR.)

Comment (6543) [reply changelog.d/16304.doc]: Well its a github issue for forks in orgs ...

Comment (DMRobertson) [reply changelog.d/16304.doc]: Not sure I follow---IIRC there is a [tickbox option for this when opening a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

In any case, it's all good now. Thanks for bearing with me!

Comment (clokep) [reply changelog.d/16304.doc]: If the fork is in an organization there's no tickbox: https://github.com/orgs/community/discussions/5634

**Pull request merged**